src/common/common.js

```diff
@@ -57,16 +57,26 @@ const common = {
     const result = new BigNumber(satoshi).div('1e8');
     return result;
   },
-  rskCoinToWeiHex(amount) {
-    const result = `0x${this.rskCoinToWei(amount).decimalPlaces(0).toString(16)}`;
+  rskCoinToWeiHex(amount, precision) {
+    const result = `0x${this.rskCoinToWei(amount, precision).decimalPlaces(0).toString(16)}`;
     return result;
   },
-  rskCoinToWei(amount) {
-    const result = new BigNumber(amount).times('1e18');
+  rskCoinToWei(amount, precision = null) {
```

Comment:
precision has to be an integer, maybe add protection using _.floor(precision)

Add comments, what are the example values of amount and precision?